account_avatax/models/account_move.py

```diff
@@ -106,6 +106,21 @@ def onchange_warehouse_id(self):
     calculate_tax_on_save = fields.Boolean()
     so_partner_id = fields.Many2one(comodel_name="res.partner", string="SO Partner")
 
+    @api.model
+    @api.depends("company_id")
+    def _compute_hide_excemption(self):
+        avatax_config = self.env.company.get_avatax_config_company()
+        for inv in self:
+            inv.hide_excemption = avatax_config.hide_excemption
+
+    hide_excemption = fields.Boolean(
+        "Hide Exemption & Tax Based on shipping address",
+        compute=_compute_hide_excemption,  # For past transactions visibility
+        default=lambda self: self.env.company.get_avatax_config_company,
```

Comment:
This is computed, so no point in having the default, AFAICS

Reply:
@dreispt computation only works when you save the record, and default gives you field value on blank record which gives better control over tax exempt field visibility